cms/envs/aws.py

```diff
@@ -392,6 +392,7 @@
     "MICROSITE_DATABASE_TEMPLATE_CACHE_TTL", MICROSITE_DATABASE_TEMPLATE_CACHE_TTL
 )
 
+FOOTER_CACHE_TIMEOUT = ENV_TOKENS.get('FOOTER_CACHE_TIMEOUT', FOOTER_CACHE_TIMEOUT)
```

Comment:
In lms/envs/aws.py this settings parameter is located in a section named 'Branded footer' -- I think it would be good to do that in this file as well, because right now it looks like it's part of the microsite configuration set and that's not really true.